.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: Run tests
+
+on: [push]
```

Comment:
I _think_ this might want to be `on: [pull_request]` or `on: [push, pull_request]` so that it runs the test on external PRs.